test/unit/sortByFilenameDateDesc.js

```diff
@@ -0,0 +1,43 @@
+
+const chai = require('chai');
+const sortByFilenameDateDesc = require('../../app/lib/sortByFilenameDateDesc');
+
+const expect = chai.expect;
+
+describe('sort by filename date', () => {
```

Comment:
Can the description be extended to include 'desc' to match what the function is doing - it just makes it a bit clearer when reading the tests output.